app/controllers/concerns/person_description_helper.rb

```diff
@@ -0,0 +1,110 @@
+  #
+  # @param person_display_name [String] Person display name
+  # @param seat_incumbency_text [String] Seat incumbency text string
+  # @return [String] String of person name with seat incumbency text
+  def self.name_with_seat_incumbency_text(person_display_name, seat_incumbency_text)
+    "#{person_display_name} #{seat_incumbency_text}."
+  end
+
+  # Returns current roles text with pronoun and seat incumbency text
+  #
+  # @param person_display_name [String] Person display name
+  # @param current_roles_text [String] Current roles text string
+  # @param person_pronoun [String] Person pronoun
+  # @param seat_incumbency_text [String] Seat incumbency text string
+  # @return [String] String of current roles text with pronoun and seat incumbency text
+  def self.current_roles_description(person_display_name, current_roles_text, person_pronoun, seat_incumbency_text)
+    "#{person_display_name} is currently #{current_roles_text}. #{person_pronoun} #{seat_incumbency_text}."
+  end
+
+  # Fetches formal body memberships from current_roles
+  #
+  # @param current_roles [Hash] Hash of current roles grouped by role type
+  # @return [Array] Array of current roles with the type 'FormalBodyMembership'
+  def self.fetch_formal_body_memberships(current_roles)
+    current_roles.fetch('FormalBodyMembership', [])
+  end
+
+  # Fetches names for current roles based on role type
+  #
+  # @param current_roles [Hash] Hash of current roles grouped by role type
+  # @return [Array] Array of strings of current role names
+  def self.fetch_current_role_names(current_roles)
+    formal_body_memberships = fetch_formal_body_memberships(current_roles)
+    current_role_names = []
+    ROLES.keys.each do |role_type|
+      current_roles.fetch(role_type, []).each do |current_role|
+        role_name = get_role_name(role_type, current_role)
+        current_role_names << role_name unless role_type == 'FormalBodyMembership'
+        current_role_names << formal_body_membership_role_name(current_role, formal_body_memberships[0], role_name) if role_type == 'FormalBodyMembership'
+      end
+    end
+    current_role_names
+  end
+
+  # Returns formal body membership role name
+  #
+  # @param current_role [Grom::Node] Role object
+  # @param first_formal_body_membership [Grom::Node] First formal body membership object
+  # @param role_name [String] Role name for formal body membership
+  # @return [String] String of formal body membership role name
+  def self.formal_body_membership_role_name(current_role, first_formal_body_membership, role_name)
+    current_role == first_formal_body_membership ? "a member of the #{role_name}" : "the #{role_name}"
+  end
+
+  # Returns role name using role type and role
+  #
+  # @param role_type [String] String of role type
+  # @param role [Grom::Node] Role object
+  # @return [String] String of role name
+  def self.get_role_name(role_type, role)
+    role.send(ROLES[role_type].to_sym).name
```

Comment:
Do we need to consider sing safe navigation here incase there is an issue and `name` is not populated?